modules/fbs/includes/fbs.availability.inc

```diff
@@ -14,11 +14,22 @@
  *   Avaliability info.
  */
 function fbs_availability_items($provider_ids) {
-  $avails = fbs_service()->Catalog->getAvailability(fbs_service()->agencyId, $provider_ids);
+  $avails = array();
+  $service = fbs_service();
+  if ($service) {
```

Comment:
Moving the fbs_service call inside the try block, also eliminates the need for the $service variable and if statement.